src/components/section/ohri-form-section.component.tsx

```diff
@@ -24,7 +24,7 @@ export const supportsUnspecified = question => {
     question.questionOptions.rendering != 'encounter-location'
   );
 };
-const OHRIFormSection = ({ fields, onFieldChange }) => {
+const OHRIFormSection = ({ sessionMode, fields, onFieldChange }) => {
```

Comment:
You don't need this, you can always access the session mode through the from context.

```js
const { encounterContext } = React.useContext(OHRIFormContext);

console.log(encounterContext.sessionMode); // prints out the session mode

```

Reply:
Cool. I actually discovered this later. Should have gone back and refactored my code. Let me do that